code/_helpers/unsorted.dm

```diff
@@ -1466,11 +1466,17 @@ var/mob/dview/dview_mob = new
 		if(337.5)
 			return "North-Northwest"
 
-
-
-
-
-
-
-
-
+#define NAMEOF(datum, X) (#X || ##datum.##X)

```

Comment:
Please add comment:
This macro is used for compile checked var-name-as-string references.   Normally if a var name is used as a string (for use in `vars` or such), A typo in the string (or the actual var being renamed in a later commit) will not generate a compile error, you'll just get runtimes.  Use this macro to make the compiler check that your string is a valid var name.